deployments/monitoring/GrafanaMatrix.Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN mkdir $APP_HOME
 WORKDIR $APP_HOME
 
 RUN git clone https://github.com/ananace/ruby-grafana-matrix.git $APP_HOME
+RUN git checkout 0d662b29633d16176291d11a2d85ba5107cf7de3
```

Comment:
Can you add a comment here? Is there something that changed and is not compatible now?

Reply:
I explained it in the commit message of this change, but I can add a comment.

tl;dr new commits to master and running it is a little different now

Reply:
> I explained it in the commit message of this change, but I can add a comment.

Reading commit messages for individual changes should not be required for a proper review, what's more - going back to the PR history to understand why a line is there in the code shouldn't be required either.

Since we squash-merge PRs `git blame` for that line is useless as well.